addons/box/README.md

```diff
@@ -3,7 +3,7 @@
 Enabling the addon for development
 
 1. If `website/addons/box/settings/local.py` does not yet exist, create a local box settings file with `cp website/addons/box/settings/local-dist.py website/addons/box/settings/local.py`
```

Comment:
I think the `website/` parts of the urls need to be taken out here.  Ditto for number 4.